plutus-core/executables/Common.hs

```diff
@@ -446,14 +488,16 @@ prettyExample =
          SomeUntypedExample (SomeUntypedTermExample (UntypedTermExample term)) ->
              PP.prettyPlcDef $ UPLC.Program () (PLC.defaultVersion ()) term
 
-toTypedTermExample :: PLC.Term PLC.TyName PLC.Name PLC.DefaultUni PLC.DefaultFun () -> TypedTermExample
+toTypedTermExample ::
+  PLC.Term PLC.TyName PLC.Name PLC.DefaultUni PLC.DefaultFun () -> TypedTermExample
 toTypedTermExample term = TypedTermExample ty term where
     program = PLC.Program () (PLC.defaultVersion ()) term
     errOrTy = PLC.runQuote . runExceptT $ do
         tcConfig <- PLC.getDefTypeCheckConfig ()
         PLC.inferTypeOfProgram tcConfig program
     ty = case errOrTy of
-        Left (err :: PLC.Error PLC.DefaultUni PLC.DefaultFun ()) -> errorWithoutStackTrace $ PP.displayPlcDef err
+        Left (err :: PLC.Error PLC.DefaultUni PLC.DefaultFun ()) ->
+          errorWithoutStackTrace $ PP.displayPlcDef err
```

Comment:
Why `errorWithoutStackTrace`?

Reply:
I'm not sure, I had the same question really. I've changed them all to `error` now.